src/state/reducers/users/index.js

```diff
@@ -42,8 +33,7 @@ export const usersReducer = createReducer(
     }),
     [USERS_FETCH_DATA_SUCCESS]: (state, payload) => ({
       ...state,
-      list: payload.list || [],
-      user: payload.user || initialState.user,
+      data: payload.data,
```

Comment:
do we need to do `data: payload.data || []` ?

Reply:
Data is set to an empty array from the initial state, so I think this is unnecesary.